provision/iam-users/iam-group.tf

```diff
@@ -0,0 +1,13 @@
+resource "aws_iam_group" "default" {
```

Comment:
can we add Read only to all other AWS resources?

Reply:
added!